store/copr/mpp.go

```diff
@@ -150,7 +152,10 @@ func (m *mppIterator) run(ctx context.Context) {
 		m.mu.Unlock()
 		m.wg.Add(1)
 		bo := backoff.NewBackoffer(ctx, copNextMaxBackoff)
-		go m.handleDispatchReq(ctx, bo, task)
+		go func(mppTask *kv.MPPDispatchRequest) {
+			m.handleDispatchReq(ctx, bo, mppTask)
+			m.wg.Done()
+		}(task)
```

Comment:
do we need to catch some errors?

Reply:
no, all the errors are sent to a channel and are handled somewhere